docs/builds/guides/frameworks/angular.md

```diff
@@ -299,6 +331,171 @@ Then, add it in the `angular.json` configuration file.
 }
 ```
 
+## Accessing the CKEditor API from an Angular Component
+
+The **ckeditor** component will provide all the functionality needed for most use cases.  In 
+cases where access to the Javascript-level {@link api CKEditor API} is needed
+it is easy to access with an additional step.
+
+To do this, assign the Angular **ckeditor** component to a locally scoped variable as
+follows:
+
+```html
+	<ckeditor #editor1 [editor]="Editor" [config]="ckconfig"
+			  [(ngModel)]="text" name="editcontent"></ckeditor>
+```
+
+You can then refer to that variable as needed in any method call within the component
+such as:
+
+```html
+	<button type="button" (click)="doPasteSomething(editor1)">Do Someting</button>
+```
+
+In the component the target function will look like this:
+
+```ts
+  doPasteSomething(editorComponent: CKEditorComponent) {
+    const editor = editorComponent.editorInstance;
```

Comment:
Note that the `editorInstance` might not be present here. That's why waiting for the `ready` event from the component is better.